python/fledge/services/core/api/support.py

```diff
@@ -191,9 +204,17 @@ async def get_syslog_entries(request):
             response['count'] = total_lines
             cmd = template.format(valid_source[source], _SYSLOG_FILE, total_lines - offset, limit)
         else:
-            cmd = non_total_template.format(valid_source[source], _SYSLOG_FILE, offset, limit)
-        a = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE).stdout.readlines()
+            scriptPath = os.path.join(_SCRIPTS_DIR, "common", "get_logs.sh")
+            # cmd = non_total_template.format(valid_source[source], _SYSLOG_FILE, offset, limit)
+            pattern = '({})\[.*\].*{}:'.format(valid_source[source], levels)
+            cmd = '{} -offset {} -limit {} -pattern \'{}\' -logfile {} -source {} -level {}'.format(scriptPath, offset, limit, pattern, _SYSLOG_FILE, source, level)
+            _logger.info('********* non_totals=true: new shell command: {}'.format(cmd))
+
+        t1 = datetime.datetime.now()
+        a = subprocess.Popen([cmd], shell=True, stdout=subprocess.PIPE).stdout.readlines()
+        t2 = datetime.datetime.now()
         c = [b.decode() for b in a]  # Since "a" contains return value in bytes, convert it to string
```

Comment:
Please rename t1, a, t2, c  variables to some more meaningful

Reply:
t1 and t2 are timestamps before and after shell script execution. I thought it would be pretty obvious.

Will change 'a' to 'rv' (return value) and 'c' to 'rv_str'?

Reply:
'a' and 'c' were already there before this PR.

Reply:
Unit test cases fail when I change 'a' to 'rv', so reverting.